apps/settings/serializers/security.py

```diff
@@ -124,6 +124,11 @@ class SecurityAuthSerializer(serializers.Serializer):
         label=_('Third-party login MFA'),
         help_text=_('The third-party login modes include OIDC, CAS, and SAML2'),
     )
+    SECURITY_MFA_BY_EMAIL = serializers.BooleanField(
+        required=False, default=False,
+        label=_('MFA via Email'),
+        help_text=_('Email as a method for multi-factor authentication')
+    )
     OTP_ISSUER_NAME = serializers.CharField(
         required=False, max_length=16, label=_('OTP issuer name'),
     )
```

Comment:
There aren't any obvious irregularities in this code snippet, but there may be room for optimization and readability improvement with regards to the use of camelCase vs `snake_case` variable names throughout the codebase. Additionally, using a consistent style sheet or commenting structure could make the code more accessible without introducing new problems into it.

However, I would need access to that specific code and context (such as its original source) to determine if these issues exist.